test/dev/async-config.size-limit.test.js

```diff
@@ -20,20 +20,20 @@ describe('nuxt basic resources size limit', () => {
   it('should stay within the size limit range in legacy mode', async () => {
     const legacyResourcesSize = await getResourcesSize(distDir, 'client', { gzip: true, brotli: true })
 
-    const LEGACY_JS_RESOURCES_KB_SIZE = 201
+    const LEGACY_JS_RESOURCES_KB_SIZE = 217
     expect(legacyResourcesSize.uncompressed).toBeWithinSize(LEGACY_JS_RESOURCES_KB_SIZE)
 
     const LEGACY_JS_RESOURCES_GZIP_KB_SIZE = 70
     expect(legacyResourcesSize.gzip).toBeWithinSize(LEGACY_JS_RESOURCES_GZIP_KB_SIZE)
 
-    const LEGACY_JS_RESOURCES_BROTLI_KB_SIZE = 60
+    const LEGACY_JS_RESOURCES_BROTLI_KB_SIZE = 64
     expect(legacyResourcesSize.brotli).toBeWithinSize(LEGACY_JS_RESOURCES_BROTLI_KB_SIZE)
   })
 
   it('should stay within the size limit range in modern mode', async () => {
     const modernResourcesSize = await getResourcesSize(distDir, 'modern', { gzip: true, brotli: true })
 
-    const MODERN_JS_RESOURCES_KB_SIZE = 172
+    const MODERN_JS_RESOURCES_KB_SIZE = 180
```

Comment:
No ideas why this increased maybe something shared? /cc @clarkdo

Reply:
polyfill？

Reply:
Indeed but it is added conditionally: https://github.com/nuxt/nuxt.js/pull/8462/files#diff-2159334eca1dc836bb579a16572dc7d1a52e7777661a52424e9efdcddb84e546R209

Reply:
I think babel detected ufo is using URL and added a polyfill

Reply:
In that case it would be worst :D Since URL constructor polyfill is only for IE (so babel preset for modern shouldn't do) and also if it was doing polyfill we didn't need it. I think only corejs@3 has this polyfill... 

Anyway nothing important since gzip is fine. Something might be wrong with tests

Reply:
corejs seems supporting URL but not URLSearchParams.

Reply:
Actually it is in [web-standards 3](https://github.com/zloirock/core-js/tree/v3#web-standards) but not [web-standards 2](https://github.com/zloirock/core-js/tree/v2#web-standards) :))

Reply:
Have you checked analyzer report?

Reply:
![image](https://user-images.githubusercontent.com/5158436/101691259-bc9ab080-3a6e-11eb-8a9e-a2e6ae558d53.png)

nothing there...

Reply:
I’ll check with my laptop

Reply:
Checked bundle size, there is no difference before and after, I think you're right there is sth wrong in test, I'll run through the code an try to fix.

Reply:
Just checked the test, I think the test is fine, actually we have a min 0.95 and max 1.05 value for the size. The max value was 172*1.05 = 180.6, the modern bundle size was around 180 which is just OK for the assertion, but the bundle size is 1-2 kb larger than before now and just exceed the max value.

Reply:
oh this makes sense now. Do you have any idea of windows failing test? (added possix due to vendor chunk creation)

Reply:
Looks this branch is not latest dev, is that expected ?

Reply:
I just merged dev and let's see the test result

Reply:
Fixed